crates/trippy-core/Cargo.toml

```diff
@@ -49,10 +49,10 @@ ipnetwork.workspace = true
 mockall.workspace = true
 rand.workspace = true
 serde = { workspace = true, default-features = false, features = [ "derive" ] }
-serde_yml.workspace = true
```

Comment:
@nc7s you can remove this from the workspace dependencies as well as it is now unused